### ISSUE TITLE
Disabled items

### ISSUE THREAD

**Issue by RichCaloggero** [microsoft/vscode#121843]
@isidorn 

This seems to show up mostly in list views.

- open run / debug (control+shift+d)
- navigate to views and actions (shows list of checkboxes containing, among others, "call stack"
- uncheck "call stack" (breakpoints also disappears from the list)

This mystified me until I realized that the state of "call stack" effected "breakpoints" as well.

I assume there is some sort of disabled state that gets applied and is removing the disabled element from the accessibility tree. This is very confusing. WIthout memorizing every option in each list, it is difficult to know when something is disabled, or whether screen reader is missing something, etc.

Is there another way of disabling something such as adding readonly state (`aria-readonly`); this leaves it in the tree, but disabled changing via the user.

**Comment by isidorn** [comment]
Unchecking call stack should not make the breakpoints disapear from the list.
So CALL STACK, BREAKPOINTS are just independet views in the debug viewlet. By unchecking you are basicaly collapsing each view, and collapsing one view does not automatically colappse the other view.

For reference here are the docs for debugging https://code.visualstudio.com/docs/editor/debugging

If you have exact steps what you do for the unexpected behavior let me know since I think there is no bug here
